Libplanet/Net/Swarm.cs

```diff
@@ -1443,6 +1452,36 @@ CancellationToken cancellationToken
                 spent);
         }
 
+        private async Task BroadcastBlockAsync(
+            TimeSpan broadcastBlockInterval,
+            CancellationToken cancellationToken)
+        {
+            const string fname = nameof(BroadcastBlockAsync);
+            while (!cancellationToken.IsCancellationRequested)
+            {
+                try
+                {
+                    await Task.Delay(broadcastBlockInterval, cancellationToken);
+                    if (BlockChain.Tip is { } tip)
+                    {
+                        BroadcastBlock(tip);
+                    }
+                }
+                catch (OperationCanceledException e)
+                {
+                    _logger.Warning(e, $"{fname}() is canceled.");
+                    throw;
+                }
+                catch (Exception e)
+                {
+                    _logger.Error(
+                        e,
+                        $"An unexpected exception occurred during {fname}(): {e}"
```

Comment:
```suggestion
                        "An unexpected exception occurred during {FName}(): {E}",
                        fname,
                        e
```
Serilog suggests that the log message is compile time constant.